pkg/audit/verifier.go

```diff
@@ -159,7 +159,7 @@ func (d *defaultDownloader) getShare(ctx context.Context, limit *pb.AddressedOrd
 	// determines number of seconds allotted for receiving data from a storage node
 	timedCtx := ctx
 	if d.minBytesPerSecond > 0 {
-		maxTransferTime := time.Duration(int32(time.Second) * bandwidthMsgSize / d.minBytesPerSecond.Int32())
+		maxTransferTime := time.Second * time.Duration(10+(bandwidthMsgSize/d.minBytesPerSecond.Int32()))
```

Comment:
This code wasn't very clear.  This morning I put a PR that simply uses int64 instead, as I was able to confirm my thoughts that this was an int32 overflow.  My new code doesn't have the +10 and still divides by ints, so I suppose it would fail if the messages was <64 bytes.  Hrmm.

Reply:
I added in a 5 second minimum check in the merge here

Reply:
cool